### PR TITLE
Inomurko/telemetry support

### PR DESCRIPTION
**Pull request by InoMurko**
0.5.6 was a mistake, should have done it with properly with telemetry!

### CODE REVIEW

File: test/ethereumex/counter_test.exs [hunk 1]
```diff
@@ -1,47 +1,65 @@
+Application.ensure_all_started(:telemetry)
```
Comment (ayrat555) [inline test/ethereumex/counter_test.exs]: I think this line should be in test_helper

File: test/ethereumex/counter_test.exs [hunk 2]
```diff
@@ -1,47 +1,65 @@
...
-      Application.put_env(:ethereumex, :adapter_options, nil)
+      :telemetry.detach(handler_id)
     end)
 
-    :ok
+    {:ok, handler_id: handler_id}
   end
 
-  test "incrementing twice returns correct number" do
+  test "incrementing twice returns correct number", _ do
```
Comment (ayrat555) [inline test/ethereumex/counter_test.exs]: I think it will work without `_`

**Comment by ayrat555** [comment]
@InoMurko https://hex.pm/packages/ethereumex/0.6.0

**Comment by InoMurko** [comment]
Thank you!

**Pull request merged**